### DISCUSSION TITLE
How do I get rid of line in the prompt theme?

### DISCUSSION THREAD

**Discussion by ryub3n** [microsoft/terminal#12566] [Q&A] [answered]
I'm using oh-my-posh and some of the themes have sections colours that overlap and are showing a strange line.
How do I get rid of these lines?
![image](https://user-images.githubusercontent.com/24279903/155550745-de131531-f05d-49cb-a472-a2aeda03744f.png)

Ideally it shouldn't show those lines like below:
![image](https://user-images.githubusercontent.com/24279903/155550942-c7feac98-0cf4-4a51-a5d2-50025bf2f46f.png)

**Comment by zadjii-msft** [comment, marked as answer]
Which font are you using? That kind of artifact can be common depending on the exact font and size that's selected. 


We're tracking this issue in #8993

**Reply by ryub3n** [reply]
I'm using FiraCode Nerd Font size 12.
Same issue if I use Caskaydia Nerd Font or change the font size.
![image](https://user-images.githubusercontent.com/24279903/155555952-341bf548-7577-4955-a512-dc3b43c6f03b.png)

**Reply by zadjii-msft** [reply]
Okay yea we're tracking this in #8993

**Reply by ryub3n** [reply]
Seems to be an issue with **ClearType**.
Switched to **Grayscale** and it appears fine.
![image](https://user-images.githubusercontent.com/24279903/155561026-f015f268-f51a-486e-8951-e238e752d639.png)
